src/Microsoft.Management.Deployment/PackageManager.cpp

```diff
@@ -424,20 +497,20 @@ namespace winrt::Microsoft::Management::Deployment::implementation
             terminationHR = queueItem->GetContext()->GetTerminationHR();
             executionStage = queueItem->GetContext()->GetExecutionStage();
         }
-        WINGET_CATCH_STORE(terminationHR);
+        WINGET_CATCH_STORE(terminationHR, APPINSTALLER_CLI_ERROR_COMMAND_FAILED);
 
         // TODO - RebootRequired not yet populated, msi arguments not returned from Execute.
         WINGET_RETURN_INSTALL_RESULT_HR(terminationHR);
     }
 
     winrt::Windows::Foundation::IAsyncOperationWithProgress<winrt::Microsoft::Management::Deployment::InstallResult, winrt::Microsoft::Management::Deployment::InstallProgress> PackageManager::InstallPackageAsync(winrt::Microsoft::Management::Deployment::CatalogPackage package, winrt::Microsoft::Management::Deployment::InstallOptions options)
     {
-        return GetInstallOperation(true, package, options);
+        return GetInstallOperation(true, package, options, nullptr);
     }
 
-    winrt::Windows::Foundation::IAsyncOperationWithProgress<winrt::Microsoft::Management::Deployment::InstallResult, winrt::Microsoft::Management::Deployment::InstallProgress> PackageManager::GetInstallProgress(winrt::Microsoft::Management::Deployment::CatalogPackage package)
+    winrt::Windows::Foundation::IAsyncOperationWithProgress<winrt::Microsoft::Management::Deployment::InstallResult, winrt::Microsoft::Management::Deployment::InstallProgress> PackageManager::GetInstallProgress(winrt::Microsoft::Management::Deployment::CatalogPackage package, winrt::Microsoft::Management::Deployment::PackageCatalogInfo catalogInfo)

```

Comment:
Should this be renamed to something more generic?  When we have other operations with progress in the future, it seems like we would want to provide a similar mechanism.

But maybe we do want to keep them separate because the Installing catalog shouldn't include uninstall (although I think it is fine for it to include "upgrade" operations).  Although we do want to serialize uninstall in the same queue.

Maybe the larger question is should it be the ActiveOperations catalog and this should be the access to that?  (Blows up the whole PR...)